ChatExchange.Net/Room.cs

```diff
@@ -300,6 +301,36 @@ public HashSet<User> GetPingableUsers()
             return users;
         }
 
+        /// <summary>
+        /// Fetches a list of all users that are currently in the room.
+        /// </summary>
+        public HashSet<User> GetCurrentUsers()
+        {
+            string html = RequestManager.Get(cookieKey, string.Format("http://chat.{0}/rooms/{1}/", Host, ID));
+            CQ doc = CQ.CreateDocument(html);
+            IDomObject obj = doc.Select("script")[3];
+            string script = obj.InnerText;
+            Regex findUsers = new Regex(@"CHAT\.RoomUsers\.initPresent\(\[(?<users>.+)\]\);", RegexOptions.Compiled | RegexOptions.Singleline);

```

Comment:
These regexes would need to be instantiated outside of the method for them to benefit from the `Compiled` flag.